src/proto_nd_flow/resources/geometry.py

```diff
@@ -426,7 +435,7 @@ def _load_charge_geometry(self):
 
         tiles = np.arange(1,len(geometry_yaml['tile_chip_to_io'])*len(det_geometry_yaml['module_to_io_groups'])+1)
         io_groups = [
-            geometry_yaml['tile_chip_to_io'][tile][chip] // 1000 * (mod-1)*2
+            geometry_yaml['tile_chip_to_io'][tile][chip] // 1000 + (mod-1)*2
```

Comment:
This was a good catch. One thing that will make it slightly better is if we don't hardcode the 2 at the end of the line (number of io_groups per module). ND-LAr will have 4 io_groups per module, and we should try to start removing as many hardcoded assumptions as we can so we can eventually have only one common `geometry.py` for all detector geometries, with all the appropriate configurations read in from the yamls.

I realize that this code is living in proto_nd_flow right now, but I think we should try to eventually make it agnostic.

Reply:
Yes, I agree we should update the hardcorded number of io_groups per module. My hope is all the source code should ideally be functional for single cube, single module, 2x2, FSD, ndlar etc...